diesel_cli/src/migrations/mod.rs

```diff
@@ -88,7 +88,7 @@ pub(super) fn run_migration_command(
                 if let Some(diff_schema) = diff_schema {
                     self::diff_schema::generate_sql_based_on_diff_schema(
                         config,
-                        matches,
+                        args,
```

Comment:
Can you verify that it continues to work passing `--database-url` as global parameter after this change? So that `diesel --database-url migration generate --diff-schema` continues to work?

Reply:
ah good spot, I'll have a look into this

Reply:
@weiznich I can confirm, the global parameters are passed to the sub-commands. I can add a test case for this if needed but it is [claps api](https://docs.rs/clap/latest/clap/struct.Arg.html#method.global) which is unlikely to change

Reply:
I think it's fine to leave these tests to clap. It's just that I forget from time to time what's the behaviour there.

Reply:
Great! I agree with leaving it to clap.